azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -158,8 +181,12 @@ def connect(self, password=None, client_certificate=None):
         self._mqtt_client.tls_insecure_set(False)
         self._mqtt_client.username_pw_set(username=self._username, password=password)
 
-        self._mqtt_client.connect(host=self._hostname, port=8883)
         self._mqtt_client.loop_start()
+        rc = self._mqtt_client.connect(host=self._hostname, port=8884)
```

Comment:
>8884 [](start = 65, length = 4)

This number is different from the one before.

Reply:
Was there a reason this was changed?

---
In reply to: [302767087](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r302767087) [](ancestors = 302767087)

Reply:
good catch.  not a clue how I changed this.  

---
In reply to: [303132850](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r303132850) [](ancestors = 303132850,302767087)